src/pages/home/home.css

```diff
@@ -0,0 +1,245 @@
+/* @import url(/components/color.css); */
+@import url('https://fonts.googleapis.com/css2?family=Roboto+Condensed&display=swap');
+@import url(https://build-ui-component-library.netlify.app/components/components.css);
+
+
+.logo{
+    cursor: pointer;
+    color:var(--pink-primary1);
+    font-size: 1.5rem;
+    padding-left: 1.5rem;
+}
+
+.logo h1 i{
+    padding-right: 1rem;
+}
+
+a {
+    text-decoration: none;
+}
+
+ul {
+    list-style: none;
+}
+
+ .header-nav {
+background-color: var(--pink-light2);
+
+}
+.header-nav{
+    position: fixed;
+    z-index:555;
```

Comment:
I think it is better not to use such high z-indexes, could become quite tough to debug once codebase grows big.

Reply:
I have updated it ,thanks for the review